test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -17,6 +17,7 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.StandardLearners\Microsoft.ML.StandardLearners.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.Onnx\Microsoft.ML.Onnx.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.TensorFlow\Microsoft.ML.TensorFlow.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.csproj" />
```

Comment:
revert whole file.

Reply:
good catch. Done

---
In reply to: [220398733](https://github.com/dotnet/machinelearning/pull/942#discussion_r220398733) [](ancestors = 220398733)

Reply:
You still have package reference on Onnx.TestModels.

---
In reply to: [220411170](https://github.com/dotnet/machinelearning/pull/942#discussion_r220411170) [](ancestors = 220411170,220398733)